app/ui/src/app/customizations/tech-extensions/tech-extensions-list.component.ts

```diff
@@ -11,6 +11,7 @@ import {
   ListEvent,
   EmptyStateConfig,
 } from 'patternfly-ng';
+import { ConfigService } from '../../config.service';
```

Comment:
Could use aliases instead to avoid the relative file paths, I'll open a separate PR for this.